wiki/faq.md

```diff
@@ -270,23 +270,33 @@ for doing so is much higher than if you were to wait for longer.
 ---
 ### Are there any fees associated with the MRC request?
 
-Yes. The following is a summary of the fees:
+Yes. The fee decreases the longer you wait between requests and helps fund 
+development. After a few MRC requests, you will probably have enough GRC to 
+stake on your own and skip MRC fees entirely. As a bonus, if you stake a 
+block with somebody else's MRC request, you collect a portion of their fee.
+
+The fee has two components:
 
 - Blockchain transaction fee: This fee is the usual fee that is required to send
 a transaction over the Gridcoin blockchain and is usually insignificant. This
 fee is charged when you make the MRC request.
 
 - MRC fee: This is the fee that is charged after the MRC request is processed
-and is applied to the rewards that you have claimed. The fee starts at 40% of
-the owed GRC at the 14-day interval since the last claim (or beacon creation)
-and decreases by `40% * 14/(number of days since last claim)`.
+and is applied to the rewards that you have claimed. The fee is equivalent to 
+roughly 5.6 days of crunching. If you request every two weeks (14 days), 5.6
+days is 40% of that. If you request every six weeks, it is 13%. 

```

Comment:
I'm not sure if the roughly 5.6 days of crunching is correct, but it's been a bit since I've looked at the formula MRC uses

Reply:
![image](https://github.com/gridcoin-community/Gridcoin-Site/assets/90811423/ffc3063f-2bf8-415d-949a-b07423ef2e7f)